modules/openapi-generator/src/test/resources/bugs/issue_13365_2.yml

```diff
@@ -0,0 +1,29 @@
+openapi: 3.0.0
+info:
+  description: Specification to reproduce nullable issue with Array
+  title: Required Api
+paths:
+  '/person':
+    post:
+      summary: Inserts a person
+      operationId: postPerson
+      tags:
+        - person
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Person'
+
+      responses: ...
+
+components:
+  schemas:
+    Person:
+      type: object
```

Comment:
just add one more model(without required fields) to above specification instead of creating separate file